src/officehours_ui/templates/base.html

```diff
@@ -3,41 +3,48 @@
 {% get_flatpages as flatpages %}
 <!doctype html>
 <html lang="en">
-  <head>
+
+<head>
```

Comment:
We'll need to clean up the commit history so that it only includes the changes relevant to the issue (#357). I guess I'd recommend creating a new branch with a different name, cherry-picking the one commit that's relevant (https://git-scm.com/docs/git-cherry-pick), deleting this branch, renaming your new branch to the old name, then force pushing to GitHub.

So something like this:
```
git checkout master
git checkout -b issue-357-temp
git cherry-pick 8f0262a027fa5db458eb4539a41f4543b147cba5
git branch -D issue-357-better-message-for-race-condition
git branch -M issue-357-better-message-for-race-condition
git push origin issue-357-better-message-for-race-condition
```